packages/@webex/internal-plugin-metrics/src/new-metrics.ts

```diff
@@ -168,6 +168,61 @@ class Metrics extends WebexPlugin {
     return this.callDiagnosticMetrics.submitClientEvent({name, payload, options});
   }
 
+  /**
+   * Submit a pre-login metric to clientmetrics
+   * @public
+   * @param payload
+   * @param preLoginId - pre-login ID of user
+   * @returns
+   */
+  public postPreLoginMetric(payload: any, preLoginId: string): Promise<any> {
+    // @ts-ignore
+    return this.webex
+      .request({
+        method: 'POST',
+        api: 'metrics',
+        resource: 'clientmetrics-prelogin',
+        headers: {
+          authorization: false,
+          'x-prelogin-userid': preLoginId,
+        },
+        body: payload,
+      })
+      .then((res) => {
+        // @ts-ignore
+        this.webex.logger.log(`Metrics: @postPreLoginMetric. Request successful:`, res);
+
+        return res;
+      })
+      .catch((err) => {
+        // @ts-ignore
+        this.logger.error(`Metrics: @postPreLoginMetric. Request failed:`, err);
+
+        return Promise.reject(err);
+      });
+  }
+
+  /**
+   * Issue request to alias a user's pre-login ID with their CI UUID
+   * @param {string} preLoginId
+   * @returns {Object} HttpResponse object
+   */
+  public clientMetricsAliasUser(preLoginId: string) {
+    // @ts-ignore
+    return this.webex.request({
+      method: 'POST',
+      api: 'metrics',
+      resource: 'clientmetrics',
+      headers: {
+        'x-prelogin-userid': preLoginId,
+      },
+      body: {},
+      qs: {
+        alias: true,
+      },
+    });
+  }
```

Comment:
shouldn't we have success and error handlers so we log whether this was or not succesfull or no?